es-model-archive/kf-es-model-v0.5.0_1.2.0/file_centric.mapping.json

```diff
@@ -0,0 +1,528 @@
+          "type": "keyword"
+        }, 
+        "is_harmonized": {
+          "type": "boolean"
+        }, 
+        "kf_id": {
+          "type": "keyword"
+        }, 
+        "latest_did": {
+          "type": "keyword"
+        }, 
+        "modified_at": {
+          "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss||yyyy-MM-dd'T'HH:mm:ss.SSSSSS+HH:mm", 
+          "type": "date"
+        }, 
+        "participants": {
+          "properties": {
+            "affected_status": {
+              "type": "boolean"
+            }, 
+            "alias_group": {
+              "type": "keyword"
+            }, 
+            "available_data_types": {
+              "type": "keyword"
+            }, 
+            "biospecimens": {
+              "properties": {
+                "age_at_event_days": {
+                  "type": "integer"
+                }, 
+                "analyte_type": {
+                  "type": "keyword"
+                }, 
+                "composition": {
+                  "type": "keyword"
+                }, 
+                "concentration_mg_per_ml": {
+                  "type": "float"
+                }, 
+                "dbgap_consent_code": {
+                  "type": "keyword"
+                }, 
+                "external_aliquot_id": {
+                  "type": "keyword"
+                }, 
+                "external_sample_id": {
+                  "type": "keyword"
+                }, 
+                "kf_id": {
+                  "type": "keyword"
+                }, 
+                "ncit_id_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "ncit_id_tissue_type": {
+                  "type": "keyword"
+                }, 
+                "sequencing_center_id": {
+                  "type": "keyword"
+                }, 
+                "shipment_date": {
+                  "type": "keyword"
+                }, 
+                "shipment_origin": {
+                  "type": "keyword"
+                }, 
+                "source_text_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "source_text_tissue_type": {
+                  "type": "keyword"
+                }, 
+                "source_text_tumor_descriptor": {
+                  "type": "keyword"
+                }, 
+                "spatial_descriptor": {
+                  "type": "keyword"
+                }, 
+                "uberon_id_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "volume_ml": {
+                  "type": "float"
+                }
+              }, 
+              "type": "nested"
+            }, 
+            "consent_type": {
+              "type": "keyword"
+            }, 
+            "diagnoses": {
+              "properties": {
+                "age_at_event_days": {
+                  "type": "integer"
+                }, 
+                "diagnosis": {
```

Comment:
This is defined in `es_mapper.yaml` but not in dataservice api, should this be removed?

Reply:
This was to support this field requested by CHOP:
https://kidsfirst.atlassian.net/wiki/spaces/KFDRC/pages/748027905/ETL+Calculations#ETLCalculations-Diagnosis

Reply:
If this is populated by the ETL, then yes, we keep it.